sdk/sql/arm-sql/CHANGELOG.md

```diff
@@ -1,15 +1,146 @@
+  - Interface ManagedInstance has a new optional parameter databaseFormat
+  - Interface ManagedInstance has a new optional parameter externalGovernanceStatus
+  - Interface ManagedInstance has a new optional parameter hybridSecondaryUsage
+  - Interface ManagedInstance has a new optional parameter hybridSecondaryUsageDetected
+  - Interface ManagedInstance has a new optional parameter isGeneralPurposeV2
+  - Interface ManagedInstance has a new optional parameter pricingModel
+  - Interface ManagedInstance has a new optional parameter storageIOps
+  - Interface ManagedInstance has a new optional parameter storageThroughputMBps
+  - Interface ManagedInstance has a new optional parameter virtualClusterId
+  - Interface ManagedInstanceUpdate has a new optional parameter authenticationMetadata
+  - Interface ManagedInstanceUpdate has a new optional parameter createTime
+  - Interface ManagedInstanceUpdate has a new optional parameter databaseFormat
+  - Interface ManagedInstanceUpdate has a new optional parameter externalGovernanceStatus
+  - Interface ManagedInstanceUpdate has a new optional parameter hybridSecondaryUsage
+  - Interface ManagedInstanceUpdate has a new optional parameter hybridSecondaryUsageDetected
+  - Interface ManagedInstanceUpdate has a new optional parameter isGeneralPurposeV2
+  - Interface ManagedInstanceUpdate has a new optional parameter pricingModel
+  - Interface ManagedInstanceUpdate has a new optional parameter storageIOps
+  - Interface ManagedInstanceUpdate has a new optional parameter storageThroughputMBps
+  - Interface ManagedInstanceUpdate has a new optional parameter virtualClusterId
+  - Interface PrivateEndpointConnection has a new optional parameter groupIds
+  - Interface ReplicationLink has a new optional parameter partnerDatabaseId
+  - Interface Server has a new optional parameter isIPv6Enabled
+  - Interface ServerUpdate has a new optional parameter isIPv6Enabled
+  - Interface SqlVulnerabilityAssessmentScanRecord has a new optional parameter lastScanTime
+  - Added Enum KnownAuthMetadataLookupModes
+  - Added Enum KnownBackupStorageAccessTier
+  - Added Enum KnownFailoverGroupDatabasesSecondaryType
+  - Added Enum KnownFreeLimitExhaustionBehavior
+  - Added Enum KnownFreemiumType
+  - Added Enum KnownHybridSecondaryUsage
+  - Added Enum KnownHybridSecondaryUsageDetected
+  - Added Enum KnownManagedInstanceDatabaseFormat
+  - Added Enum KnownMinimalTlsVersion
+  - Added Enum KnownPhase
 
-### Other Changes
+### Breaking Changes
 
+  - Type of parameter provisioningState of interface ManagedInstance is changed from ManagedInstancePropertiesProvisioningState to ProvisioningState
+  - Type of parameter provisioningState of interface ManagedInstanceUpdate is changed from ManagedInstancePropertiesProvisioningState to ProvisioningState
+  - Type of parameter intervals of interface QueryStatisticsProperties is changed from QueryMetricInterval[] to QueryMetricIntervalAutoGenerated[]
```

Comment:
there exists duplicated models? could you check this?

Reply:
this is introduced by mixed apiversion
https://github.com/Azure/sdk-release-request/issues/5379#issuecomment-2333179440